my-app/src/app/core/store/app.state.ts

```diff
@@ -0,0 +1,43 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
+import { Course } from '../services/course/model/course';
+import * as auth from './auth/reducers/auth.reducers';
+import * as course from './courses/reducers/course.reducers';
+
+export interface AppState {
+  authState: auth.AuthState;
+  courseState: course.CourseState;
+}
+
+export const reducers = {
+  auth: auth.reducer,
+  course: course.reducer,
+};
+
+export const authFeatureSelector = createFeatureSelector<auth.AuthState>(
+  'auth'
+);
+export const authToken = createSelector(
```

Comment:
selectors for each feature should be in separate file on the same level as reducer and actions.
For example, for authState you should have
auth.selectors.ts
Where you should have all selectors related to auth state